src/agent/Shared/Fundamentals/AbortHandler.h

```diff
@@ -38,7 +36,7 @@ namespace Fundamentals {
 
 
 struct AbortHandlerConfig {
-	static const unsigned int MAX_DIAGNOSTICS_DUMPERS = 5;
+	static constexpr unsigned int MAX_DIAGNOSTICS_DUMPERS = 5;
```

Comment:
why constexpr for a numeric literal?

Reply:
So the compiler doesn't need to create a symbol and can optimize better. constexpr is basically always better than const if you plan to use it as a constant. It works like a better #define.